cms-adapter/src/Rsbc.Dmf.CaseManagement/Manager/DocumentManager.cs

```diff
@@ -253,15 +254,17 @@ public async Task<LegacyDocument> GetLegacyDocument(string documentId)
         public IEnumerable<Document> GetDocumentsByTypeForUsers(IEnumerable<Guid> loginIds, string documentTypeCode)
         {
             var documents = new List<bcgov_documenturl>();
+
             foreach (var loginId in loginIds)
             {
+
                 documents.AddRange(dynamicsContext.bcgov_documenturls
                     .Expand(doc => doc.dfp_DocumentTypeID)
                     .Expand(doc => doc.bcgov_CaseId)
                     .Expand(doc => doc.bcgov_CaseId.customerid_contact)
                     .Where(doc => 
                         doc.dfp_DocumentTypeID.dfp_code == documentTypeCode
-                        && (doc._dfp_loginid_value != null && doc._dfp_loginid_value.Value == loginId)
+                        && doc.dfp_LoginId.dfp_loginid == loginId
```

Comment:
Is there a reason for changing this logic?  The underscore property should be the more efficient way

Reply:
In this case the _property was not working as expected so has to change to use the  dfp_login lookup